interface/src/Application.cpp

```diff
@@ -7530,6 +7532,10 @@ void Application::registerScriptEngineWithApplicationServices(const ScriptEngine
     scriptEngine->registerGlobalObject("HifiAbout", AboutUtil::getInstance());  // Deprecated.
     scriptEngine->registerGlobalObject("ResourceRequestObserver", DependencyManager::get<ResourceRequestObserver>().data());
 
+    
+    scriptEngine->registerGlobalObject("ExternalResource", ExternalResource::getInstance());
+    scriptEngine->registerEnum("ExternalResource.Bucket", QMetaEnum::fromType<ExternalResource::Bucket>());
```

Comment:
if we do it this way, you need to type "ExternalResource.Bucket.Blah" every time, vs. just "ExternalResource.Blah".  I don't have an opinion either way, but some people might vote for the shorter one, while some people might prefer the explicitness of the former.

Reply:
Ok, changed